siphon/tests/test_acis.py

```diff
@@ -0,0 +1,54 @@
+"""Test ACIS Web Services API Access."""
+from siphon.simplewebservice.acis import acis_request
+from siphon.testing import get_recorder
+
+recorder = get_recorder(__file__)
+
+
+@recorder.use_cassette('acis_request')
+def test_acis_metadata():
+    """Testing ACIS MetaData request."""
+    data = acis_request('StnMeta', {'sids': 'KLNK'})
+
+    assert data['meta'][0]['uid'] == 12527
+
+
+def test_acis_stndata():
+    """Testing ACIS Station Data request."""
+    data = acis_request('StnData', {'sid': 'klnk', 'elems': [
+                        {'name': 'avgt', 'interval': 'dly'},
+                        {'name': 'mint', 'interval': 'dly'}], 'date': '20000101'})
+
+    assert data['meta']['uid'] == 12527
+    assert data['data'][0][0] == '2000-01-01'
+    assert data['data'][0][1] == '37.5'
+    assert data['data'][0][2] == '26'
+
+
+def test_acis_multistn():
+    """Testing ACIS Multi Station Data request."""
+    data = acis_request('MultiStnData', {'sids': 'klnk,kgso', 'elems': [
+                        {'name': 'avgt', 'interval': 'dly'},
+                        {'name': 'mint', 'interval': 'dly'}], 'date': '20000101'})
+
+    assert data['data'][0]['meta']['uid'] == 12527
+    assert data['data'][0]['data'][0] == '37.5'
+    assert data['data'][1]['meta']['uid'] == 13284
+    assert data['data'][1]['data'][0] == '49.0'
+
+
+def test_acis_griddata():
+    """Testing ACIS Gridded Data request."""
+    data = acis_request('GridData', {'loc': '-95.36, 29.76', 'sdate': '2000-01',
+                        'edate': '2000-07', 'grid': '3', 'elems': [
+                        {'name': 'maxt', 'interval': 'mly', 'reduce': 'max',
```

Comment:
I'm unsure of how to take care of this PEP8 error. I thought line 44 might have been too long, but instead I created the E128 error.
`siphon/tests/test_acis.py:44:25: E122 continuation line missing indentation or outdented
siphon/tests/test_acis.py:45:25: E128 continuation line under-indented for visual indent`

Reply:
Try this:
```python
    data = acis_request('GridData',
                        {'loc': '-95.36, 29.76', 'sdate': '2000-01', 'edate': '2000-07',
                         'grid': '3', 'elems': [{'name': 'maxt', 'interval': 'mly',
                                                 'reduce': 'max', 'smry': 'max'}]})
```
Arguments to the function, keys in the dictionary, items in the list--they line up when wrapped.